docs/slides/content/tech-exercise-iii.md

```diff
@@ -136,3 +139,4 @@ Answering the age old question of how do i know my tests are any good??
 <!-- .slide: data-background-image="images/chef-background.png", class="white-style" -->
 ### Related & Used Practices
 - [](https://openpracticelibrary.com/practice/) - 
+<!--TechEdit; this displays as an empty list with one bullet and a dash; need to provide the link text for the existing bullet.-->
```

Comment:
This this should be 

- [Open Practice Library]((https://openpracticelibrary.com/practice/))